include/qore/QoreCounter.h

```diff
@@ -59,7 +59,11 @@ class QoreCounter {
    DLLEXPORT void destructor(ExceptionSink* xsink);
 
    //! increments the counter
-   DLLEXPORT void inc();
+   /** @return the current value after the increment
+
+       @since %Qore 0.8.13 the current value is returned
```

Comment:
Will this be a part of Qore 0.8.13.9 or should this say 0.8.14 or maybe something else? Thanks.

Reply:
yes, 0.8.13.9

Reply:
Thanks